airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -15,6 +15,23 @@ create table "public"."actor"(
   constraint "actor_pkey"
     primary key ("id")
 );
+create table "public"."actor_catalog"(
+  "id" uuid not null,
+  "catalog" jsonb not null,
+  "catalog_hash" varchar(60) not null,
+  "created_at" timestamptz(35) not null,
+  constraint "actor_catalog_pkey"
+    primary key ("id")
+);
+create table "public"."actor_catalog_fetch_event"(
+  "id" uuid not null,
+  "actor_catalog_id" uuid not null,
+  "actor_id" uuid not null,
```

Comment:
I think it would make sense to do an index on `actor_id`. I think the most common query we will do on this table is get the record for an actor or get all records for the actor. Potentially

Reply:
Done